superset/db_engine_specs.py

```diff
@@ -501,11 +520,26 @@ def convert_dttm(cls, target_type, dttm):
             """TO_TIMESTAMP('{}', 'YYYY-MM-DD"T"HH24:MI:SS.ff6')"""
         ).format(dttm.isoformat())
 
+    @staticmethod
+    def mutate_label(label):
+        """
+        Oracle 12.1 and earlier support a maximum of 30 byte length object names, which
```

Comment:
OMG they finally fixed this in Oracle!!!!!!!!!!!!!!! I thought this day would never happen.